### PR TITLE
Update issue templates

### PR DESCRIPTION
**Pull request by volkodavs**

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/feature_request.md [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+name: Feature request
+about: Suggest an idea for this project
+title: ''
+labels: ''
+assignees: ''
+
+---
+
+**Is your feature request related to a problem? Please describe.**
+A clear and concise description of what the problem is. Ex. I'm always frustrated when [...]
```
Comment (alessandrodalbello) [inline .github/ISSUE_TEMPLATE/feature_request.md]: "E.g." instead of "Ex"?

File: .github/ISSUE_TEMPLATE/bug_report.md [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+A clear and concise description of what the bug is.
+
+**Code**
+Code snippet 
+
+**Expected behavior**
+A clear and concise description of what you expected to happen.
+
+**Actual Behavior**
+A clear and concise description of what is actual happen
```
Comment (alessandrodalbello) [inline .github/ISSUE_TEMPLATE/bug_report.md]: "...is actually happening".

File: .github/ISSUE_TEMPLATE/bug_report.md [hunk 3]
```diff
@@ -0,0 +1,26 @@
...
+Code snippet 
+
+**Expected behavior**
+A clear and concise description of what you expected to happen.
+
+**Actual Behavior**
+A clear and concise description of what is actual happen 
+
+**Stacktrace**
+Stacktrace output
```
Comment (alessandrodalbello) [inline .github/ISSUE_TEMPLATE/bug_report.md]: Please, close sentences with a period.

**Comment by volkodavs** [comment]
Yeah, this is a standard way to make an issue template

**Comment by volkodavs** [comment]
In the next following PR i will adjust codacy template to ignore some of the code

**Pull request merged**